common/formatting/unwrapped_line.h

```diff
@@ -38,6 +38,8 @@ enum class PartitionPolicyEnum {
   // There's no kNeverExpand, because one would just not create the partition in
   // the first place.
 
+  kAppendFittingSubpartitions, // all or nothing
```

Comment:
Could you elaborate on "all or nothing"?

Reply:
Done.